onnxruntime/python/tools/tensorrt/perf/post.py

```diff
@@ -0,0 +1,188 @@
+import argparse
+import mysql.connector
+import sys
+import os
+import subprocess
+import pandas as pd
+from sqlalchemy import create_engine
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-r", "--report_folder", help="Path to the local file report", required=True)
+    parser.add_argument(
+        "-c", "--commit_hash", help="Commit id", required=True)
+    parser.add_argument(
+        "-u", "--report_url", help="Report Url", required=True)
+    
+    return parser.parse_args()
+
+def parse_csv(report_file):
+    table = pd.read_csv(report_file)
+    return table
+
+def insert_latency(commit_hash, report_url, latency): 
+    
+    # connect to database
+    cnx = mysql.connector.connect(
+        user='powerbi@onnxruntimedashboard',
+        password=os.environ.get('DASHBOARD_MYSQL_ORT_PASSWORD'),
+        host='onnxruntimedashboard.mysql.database.azure.com',
```

Comment:
maybe better to put all the constant string values in one location (maybe at the top) that can be changed together?
e.g. i see onnxruntimedashboard.mysql.database.azure.com string multiple times in this file.
better if we don't need to modify multiple lines to change a constant.